src/render.tsx

```diff
@@ -136,7 +136,7 @@ function updateWithAct(
   };
 }
 
-interface DebugFunction {
```

Comment:
You probably need to add it to package public exports in `pure.ts`

Reply:
I've added the type DebugFunction to the `pure.ts` export types in my last commit. That's what you meant?